RussianFilter/sections/css_extended.txt

```diff
@@ -217,6 +217,9 @@ list-org.com#$?#.menu > div[class="dmenu sticky"]:has(> center > ins.adsbygoogle
 reporter-ua.com#?#.row > .sidebar > .block:has(> h4:contains(mgid))
 4mama.ua#?#.sidebar a.sb-projects-header-lbl:contains(Новости партнеров)
 mysitem.ru#?#.custom-none:has(> div[id^="yandex_rtb"])
+cikavosti.com#?#aside > .widget:has(noindex > a[target="_blank"][rel="nofollow"])
+cikavosti.com#?#aside > .widget:has(> .widget-top:contains(Реклама))
+cikavosti.com#?#.a-single > div[id]:has(a[target="_blank"][rel="nofollow"][href^="//goods.redtram.com"])
```

Comment:
`[target="_blank"][rel="nofollow"]` is redundant

Reply:
@Alex-302,
deleted redundant part of the rule

Reply:
I mean this part `:has(a[target="_blank"][rel="nofollow"][href^="//goods.redtram.com"])`

Reply:
fixed